light-client/tests/support/model_based/MC4_4_faulty.tla

```diff
@@ -6,10 +6,40 @@ TARGET_HEIGHT == 4
 TRUSTING_PERIOD == 1400 \* two weeks, one day is 100 time units :-)
 IS_PRIMARY_CORRECT == FALSE
 
+\* @typeAlias: BLOCKHEADER = [height: Int, time: Int, lastCommit: Set(Str), VS: Set(Str), NextVS: Set(Str)];
+\* @typeAlias: BLOCK = [header: BLOCKHEADER, Commits: Set(Str)];
+\* @typeAlias: BLOCKSTATUS = Int -> Str;
+\* @typeAlias: BLOCKS = Int -> BLOCK;
+MC4TypeAliases == TRUE
+
 VARIABLES
```

Comment:
@konnov is there a reasonable way to avoid having to repeat these annotations here?

I tired extending the Lightclient003 module, but that caused operator def conflicts.

Reply:
I think the hack by Ilina was to refactor these repetitive definitions in a separate module like `TypeDefs` and extend that one.

Reply:
Ah! Clever. We should do that, but I'll wait until we've consolidated the specs before a more invasive refactoring. Thanks! :)